### ISSUE TITLE
Merge News and Press pages

### ISSUE THREAD

**Issue by chalin** [Dindihub/docsy-starter#15]
After #13 is merged, consider making the following changes:

- Add a `## Press` header to the **News** page
- Move the press page content to the end of the **News** page
- Delete the press page file.
- Add the top of the **News** page, add a header with some explanation, something like this:
  ```markdown
  ## Highlights
  
  This section contains news highlights. For a more complete list, see [Press](#press).
  ```
- Add `/press` to the list of `aliases` in the front matter.

@lukpueh - WDYT?

**Comment by lukpueh** [comment]
Sounds good!

**Comment by chalin** [comment]
Closed by #17
